fastlane/swift/LaneFileProtocol.swift

```diff
@@ -52,7 +48,11 @@ public class LaneFile: NSObject, LaneFileProtocol {
     private static var laneFunctionNames: [String] {
         var lanes: [String] = []
         var methodCount: UInt32 = 0
-        let methodList = class_copyMethodList(self, &methodCount)
+        #if !SWIFT_PACKAGE
+            let methodList = class_copyMethodList(self, &methodCount)
+        #else
+            let methodList = class_copyMethodList(type(of: fastfileInstance!), &methodCount)
+        #endif
```

Comment:
@minuscorp Can we put a comment above this on why the packaged version is different? 😇